devel/valgrind/Portfile

```diff
@@ -129,3 +129,20 @@ subport valgrind-devel {
 
     livecheck.type none
 }
+
+subport valgrind-macos-devel {
+    epoch           1
```

Comment:
This is a new (sub)port, so it does not need the epoch.

```suggestion
```

This (sub)port's name is unusual, since it is self-understood that all ports in MacPorts are intended to work on macOS.

What is the story with this valgrind fork? If it is scheduled to be merged into the mainline valgrind soon, we might just wait and update valgrind-devel to that version at that time. If it is not scheduled to be merged soon, what's wrong with it, and might that be a reason for us not to include it?

Reply:
@ryandesign this repository is series of patches that was offered to upstream here: https://bugs.kde.org/show_bug.cgi?id=399584

Because this issue existed for quite a while I'd like to say that upstream team doesn't seem to be much appetite/time for macOS support.

For me this repository looks like as resabale tradeoff because right now valgrind is completely useless on new OS.

About the name: can you suggest something? I'm run out any ideas and agree that current name quite odd.